LondonBritishLibrary/orient/BLorient12996.xml

```diff
@@ -0,0 +1,292 @@
+                                </msItem>
+                                <msItem xml:id="p1_i1.12">
+                                    <locus from="22v"/>
+                                    <title ref="LIT00000000000000000000000#ParticipleAgentis">De Participijs Agentibus et Patientibus</title>
+                                    <note>Presumably on participles that name the actor and those that name the person affected by an action (i.e. active 
+                                        and passive participles).</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.13">
+                                    <locus from="26r"/>
+                                    <title ref="LIT00000000000000000000000#DeclensionParticiples">Declinatio supradictorum duorum Participii</title>
+                                    <note>Declension of the two participles above.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.14">
+                                    <locus from="28v"/>
+                                    <title ref="LIT00000000000000000000000#VerbalNoun">De Abstracto seu Nomine Verbali</title>
+                                    <note>On the verbal noun e.g. the gerund.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.15">
+                                    <locus from="32r"/>
+                                    <title ref="LIT00000000000000000000000#TablePronounAffixes">Tabula Pronominum Verbis affixorum</title>
+                                    <note>Table showing the pronominal affixes as they appear on the verb.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.16">
+                                    <locus from="32v"/>
+                                    <title ref="LIT00000000000000000000000#TablePresent">Seconda Tabula pro Tempore Praesenti</title>
+                                    <note>Second table shows the verbal forms of the present tense.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.17">
+                                    <locus from="34r"/>
+                                    <title ref="LIT00000000000000000000000#Negative">Conjugazio Verbi Negativi, Modo Arabico</title>
+                                    <note>Conjugation (sic) of negated verbs in the Arabic mode (sic).</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.18">
+                                    <locus from="35v"/>
+                                    <title ref="LIT00000000000000000000000#Dicere">Conjug. Verbi Affirmativi Dicere in Amahara</title>
+                                    <note>On the conjugation of affirmative verbs with a form of 'to say' (<foreign xml:lang="am">በል፡</foreign>).</note>
```

Comment:
Maybe I misunderstood the Latin, but as I see it, this section is about the conjugation of the verb "to say". I don't understand why the imperative በል given.